cosmpy/aerial/tx.py

```diff
@@ -172,19 +233,61 @@ def add_message(self, msg: Any) -> "Transaction":
         self._msgs.append(msg)
         return self
 
+    def online_seal(
```

Comment:
This method can be extended to be able to use multiple senders - it would require multiple senders and accounts. If we do that we should consider where to store queried accounts to avoid unnecessary querying.